gitclone/checkout.go

```diff
@@ -228,3 +233,17 @@ func selectFallbacks(checkoutStrategy CheckoutMethod, fetchOpts fetchOptions) fa
 		return nil
 	}
 }
+
+func createManualMergeParams(cfg Config) (*PRManualMergeParams, *ForkPRManualMergeParams, error) {
```

Comment:
this could be simplified by using named return values, something like:

```
func createManualMergeParams(cfg Config) (prManualMergeParam *PRManualMergeParams, forkPRManualMergeParam *ForkPRManualMergeParams, err error) {
	if isFork(cfg.RepositoryURL, cfg.PRRepositoryURL) {
 		forkPRManualMergeParam, err = NewForkPRManualMergeParams(cfg.Branch, cfg.PRRepositoryURL, cfg.BranchDest)
 	} else {
 		prManualMergeParam, err = NewPRManualMergeParams(cfg.Branch, cfg.Commit, cfg.BranchDest)
 	}
       return
}
```

Reply:
You're right!